### PR TITLE
9497 identify consolidated cases in qc display

### PR DESCRIPTION
**Pull request by rgoshen**
This PR addresses story [9497](https://github.com/flexion/ef-cms/issues/9497).  This adds either a "Lead case" icon and tool tip or "Consolidated case" icon and tool tip to all work items part of a consolidated group in Document QC for docket clerks.

### CODE REVIEW

File: shared/src/business/useCases/editDocketEntry/completeDocketEntryQCInteractor.js [hunk 1]
```diff
@@ -131,6 +135,10 @@ exports.completeDocketEntryQCInteractor = async (
       editState: '{}',
       relationship: DOCUMENT_RELATIONSHIPS.PRIMARY,
       userId: user.userId,
+      workItem: {
```
Comment (mrinsin) [inline shared/src/business/useCases/editDocketEntry/completeDocketEntryQCInteractor.js:138]: Could this be unit tested?

File: web-client/integration-tests/journey/docketClerkViewsSectionInboxNotHighPriority.js [hunk 2]
```diff
@@ -16,3 +22,109 @@ export const docketClerkViewsSectionInboxNotHighPriority = cerebralTest => {
     expect(inProgressWorkItem.trialDate).toBeFalsy();
   });
 };
+
+// const getInboxQueue = async cerebralTest => {
```
Comment (mrinsin) [inline web-client/integration-tests/journey/docketClerkViewsSectionInboxNotHighPriority.js:26]: Nit: Not sure if these comments were left here intentionally!

File: web-client/integration-tests/journey/petitionsClerkViewsSectionDocumentQC.js [hunk 3]
```diff
@@ -23,6 +23,8 @@ export const petitionsClerkViewsSectionDocumentQC = (
       });
       cerebralTest.petitionsClerkSectionDocumentQCInboxCount =
         helper.sectionInboxCount;
+
+      console.log('sectionInbox', helper.sectionInboxCount);
```
Comment (mrinsin) [inline web-client/integration-tests/journey/petitionsClerkViewsSectionDocumentQC.js:27]: accidental log left behind!

File: shared/src/business/useCases/docketEntry/addPaperFilingInteractor.js [hunk 4]
```diff
@@ -107,6 +107,7 @@ exports.addPaperFilingInteractor = async (
       docketNumberWithSuffix: caseToUpdate.docketNumberWithSuffix,
       inProgress: isSavingForLater,
       isRead: user.role !== ROLES.privatePractitioner,
+      leadDocketNumber: caseToUpdate.leadDocketNumber,
```
Comment (mrinsin) [inline shared/src/business/useCases/docketEntry/addPaperFilingInteractor.js:110]: Unit test?

**Review by mrinsin — APPROVED** [review]
Some missing unit tests + cleanup but otherwise, looks great!

File: web-client/integration-tests/journey/petitionsClerkViewsSectionDocumentQC.js (hunk 3 above)
Comment (rgoshen) [reply web-client/integration-tests/journey/petitionsClerkViewsSectionDocumentQC.js:27]: Thanks removed

**Comment by rgoshen** [comment]
Too many files I did not change showing up in PR

**Pull request closed without merging**